pkg/util/file.go

```diff
@@ -0,0 +1,58 @@
+/*
+Copyright 2016 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import (
+	"os"
+	"path/filepath"
+)
+
+// isDirectory checks if a path points to a directory
+func isDirectory(path string) (bool, error) {
+	stat, err := os.Stat(path)
+	if err != nil {
+		return false, err
+	}
+
+	return stat.IsDir(), nil
+}
+
+// ExpandFiles is responsible for resolving any references to directories
+func ExpandFiles(path string) ([]string, error) {
```

Comment:
I think there is some code in kubectl that we should probably reuse. I can check tomorrow.

@justinsb thoughts?

Reply:
same as above